electrum/submarine_swaps.py

```diff
@@ -194,9 +216,31 @@ async def _claim_swap(self, swap: SwapData) -> None:
             swap.funding_txid = txin.prevout.txid.hex()
             swap._funding_prevout = txin.prevout
             self._add_or_reindex_swap(swap)  # to update _swaps_by_funding_outpoint
+            funding_height = txin.block_height
             spent_height = txin.spent_height
+
+            if swap.is_reverse and swap.preimage is None:
+                if funding_height <= 0:
+                    continue
```

Comment:
`txin.block_height` is not SPV-ed.
Tou can do something like `adb.get_tx_height(txin.prevout.txid.hex()).conf >= 1`

Reply:
fixed, thanks